common/src/java/com/zimbra/common/io/TcpServerInputStream.java

```diff
@@ -50,18 +57,23 @@ public TcpServerInputStream(InputStream in, int size) {
      *         the end of the stream has been reached
      * @throws IOException
      */
-    public String readLine() throws IOException {
-        buffer.delete(0, buffer.length());
-        while (true) {
-            int ch = read();
-            if (ch == -1) {
-                return null;
-            } else if (ch == CR) {
-                continue;
-            } else if (ch == LF) {
-                return buffer.toString();
-            }
-            buffer.append((char)ch);
-        }
-    }
+	public String readLine() throws IOException {
+		buffer.delete(0, buffer.length());
+		while (true) {
+			int ch = br.read();
+			if (ch == -1) {
+				return null;
+			} else if (ch == CR) {
+				continue;
+			} else if (ch == LF) {
+				return buffer.toString();
+			}
+			buffer.append((char)ch);
+		}
+	}
+
+	 public void close ()  throws IOException {
+        br.close();
```

Comment:
there seem to be a lot of inconsistencies on spaces vs tabs throughout your changes. It's more noticeable in some spots than others because of how the code lines up (or doesn't) - here in particular it's very noticeable because. Please fix in all relevant locations